src/ILVerify/src/AssemblyInfo.cs

```diff
@@ -0,0 +1,3 @@
+﻿using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("ILVerify.Tests")]
```

Comment:
Would it make sense for the test to generate the method tokens to verify and pass them to the verification method in the ILVerify?

It should avoid the need to have access to the CoreRT type system from the test driver; and it will work towards a .dll flavor of the verifier.